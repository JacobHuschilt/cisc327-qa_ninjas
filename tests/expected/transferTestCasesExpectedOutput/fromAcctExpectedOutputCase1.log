OK. First account verified in agent mode.