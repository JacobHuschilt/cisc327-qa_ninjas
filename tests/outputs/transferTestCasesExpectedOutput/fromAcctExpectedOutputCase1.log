first account verified in agent mode