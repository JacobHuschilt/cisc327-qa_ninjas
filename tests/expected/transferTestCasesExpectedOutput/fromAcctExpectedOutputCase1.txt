XFR 3490500 1234567 25000
EOS 0000000 000 0000000 ***